Makefile

```diff
@@ -29,18 +29,19 @@ build: ## Builds the all docker services.
 	docker compose build
 
 run-devserver: # Runs the dev-server
-	docker compose up dev-server -d
+	docker compose up dev-server -d --no-deps
 	$(MAKE) update.requirements
+	docker compose up angular-build -d
 	$(MAKE) update.package
 	$(MAKE) run-offline
 
 run-offline: # Runs the dev-server in offline mode
 	docker compose up dev-server -d
 	@printf 'Please wait while the development server starts...\n\n'
-	@while [[ $$(curl -s -o /tmp/status_code.txt -w '%{http_code}' http://localhost:8181/community-library) != "200" ]]; do \
+	@while [[ $$(curl -s -o /tmp/status_code.txt -w '%{http_code}' http://localhost:8181) != "200" ]] || [[ $$(curl -s -o /tmp/status_code.txt -w '%{http_code}' http://localhost:8181/community-library) != "200" ]]; do \
```

Comment:
Can you explain the change here?

Reply:
this is required in order to check if angular-build and webpack-compiler processes are completed behind the scenes. we are checking it by checking the status code from localhost:8181 and localhost:8181/community-library

Reply:
The symbol used here `||` is the logical OR. I think you want to use the logical AND symbol

Reply:
no, that should be logical OR only, because we want to check if any of the page doesnt return status code 200, then sleep for 5 seconds. ` != "200"` I am checking "not equal to" 200